src/event-cache/EventCache.ts

```diff
@@ -75,11 +75,11 @@ export class EventCache {
      * Add an event to the cache and reset the session timer.
      *
      * If the session is being recorded, the event will be recorded.
-     * If the session is not being recorded, the event will not be recorded.
+     * If the session is not being recorded, the event will not be recorded
      *
      * @param type The event schema.
      */
-    public recordEvent = (type: string, eventData: object) => {
+    public recordEvent = (type: string, eventData = {}) => {
```

Comment:
nit: Missing type

Reply:
From what I saw, default values declares the type as well: https://www.typescriptlang.org/docs/handbook/functions.html - The reason for adding a default value was due to this comment: https://github.com/aws-observability/aws-rum-web/pull/188#discussion_r937359770

Reply:
Actually, just tested, it does not declare the type explicitly. Will be making changes